### DISCUSSION TITLE
Using a standalone configuration with a remote sensor via IPSEC

### DISCUSSION THREAD

**Discussion by pimpernoni** [Security-Onion-Solutions/securityonion#2929] [Unsupported Versions]
I  am working through the setup for SO 2.3 using VBox prototyping in anticipation of the EOL for 16.4. I have been previously using a single standalone 16.4 box on our SOHO network, and recently added a remote network connected via IPSEC.

I am able to send remote network syslog data via the IPSEC, but it cannot get through to the SO box, even though pings pass through the network to SO and the so-allow opens connections to SO from the remote network.

As a workaround, I sent remote syslogs to get repackaged at the local firewall (a pfSense box) on the subnet where SO resides using syslog-ng and then send those to SO via UDP514. However,  useful data to distinguish the origin and service, such as firewall events, kernal information and snort warnings are imbedded in the repackaging and do not parse properly in the SO Kibana interface. Everything shows up as coming from the IPSEC remote gateway with the unparsed packet detail only available in the pcap.

I am assuming that I need to set up a sensor on the remote network to process content to the manager node, but I have a small network that is easily handled by my current SO box with 6 core i-5 CPU, 32 GB RAM and 1 TB Raid storage for /nsm files. I do not want to have a separate manager and sensing node on the main SOHO network. Is there a configuration in SO2.3  whereby a standalone deployment can have a remote sensor added?

Thanks
P

**Comment by TOoSmOotH** [comment]
Is there NAT involved?

**Comment by pimpernoni** [comment]
Yes Mike, I have a small SOHO, and will be sending data via IPSEC already
established and working for syslogs

Thanks

P

**Reply by TOoSmOotH** [reply]
2.3.30 should allow you to use VPN. You cannot port forward the Manager unless you disable full verification in the filebeat config.
